metadata_service/proxy/neo4j_proxy.py

```diff
@@ -816,6 +816,46 @@ def get_latest_updated_ts(self) -> Optional[int]:
         else:
             return None
 
+    @timer_with_counter
+    def get_statistics(self) -> Dict[str, Any]:
+        """
+        API method to fetch statistics metrics for neo4j
+        :return: dictionary of statistics
+        """
+        query = textwrap.dedent("""
+        MATCH (table_node:Table) with count(table_node) as number_of_tables
+        MATCH p=(item_node)-[r:DESCRIPTION]->(description_node)
+        WHERE size(description_node.description)>2 and exists(item_node.is_view)
+        with count(item_node) as number_of_documented_tables, number_of_tables
+        MATCH p=(item_node)-[r:DESCRIPTION]->(description_node)
+        WHERE  size(description_node.description)>2 and exists(item_node.sort_order)
+        with count(item_node) as number_of_documented_cols, number_of_documented_tables, number_of_tables
+        MATCH p=(table_node)-[r:OWNER]->(user_node) with count(distinct table_node) as number_of_tables_with_owners,
+        count(distinct user_node) as number_of_owners, number_of_documented_cols,
+        number_of_documented_tables, number_of_tables
+        MATCH (item_node)-[:DESCRIPTION]->(description_node)
+        WHERE  size(description_node.description)>2 and exists(item_node.is_view)
+        MATCH  (item_node)-[:OWNER]->(user_node)
+        with count(item_node) as number_of_documented_and_owned_tables,
+        number_of_tables_with_owners, number_of_owners, number_of_documented_cols,
+        number_of_documented_tables, number_of_tables
+        Return number_of_tables, number_of_documented_tables, number_of_documented_cols,
+        number_of_owners, number_of_tables_with_owners, number_of_documented_and_owned_tables
+        """)
+        LOGGER.info('Getting Neo4j Statistics')
+        records = self._execute_cypher_query(statement=query,
+                                             param_dict={})
+        for record in records:
+            neo4j_statistics = {'number_of_tables': record['number_of_tables'],
+                                'number_of_documented_tables': record['number_of_documented_tables'],
+                                'number_of_documented_cols': record['number_of_documented_cols'],
+                                'number_of_owners': record['number_of_owners'],
+                                'number_of_tables_with_owners': record['number_of_tables_with_owners'],
+                                'number_of_documented_and_owned_tables': record['number_of_documented_and_owned_tables']
+                                }
+            return neo4j_statistics
```

Comment:
interesting, @allisonsuarez @Golodhros do you think we should include it in the browse page?

Reply:
Do we have any thoughts/updates on this one since it has been handing for quite some time.
Thanks.